### PR TITLE
📦 NEW: Config.yml generator + setup latest Go sites

### PR DESCRIPTION
**Pull request by curtisbelt**

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -64,75 +46,67 @@ sites:
         AUTOMATIC_UPDATER_DISABLED: true
         DISALLOW_FILE_EDIT: true
         DISALLOW_FILE_MODS: true
-        WP_DEBUG: true
+        WP_DEBUG: false
```
Comment (ethitter) [inline config.yml:49]: Another option is to leave this enabled and set `WP_DEBUG_DISPLAY` to `false`. We already set `WP_DEBUG_LOG`, so our concern is the front-end output.

Comment (brandoncamenisch) [reply config.yml:49]: This was on by default because we know that very few will ever bother to turn it on.

Comment (curtisbelt) [reply config.yml:49]: @ethitter Thank you -- I didn't realize WP_DEBUG_LOG required WP_DEBUG to work!
@brandoncamenisch Is this OK? I just feel everyone is probably turning this off right away anyway, as there's so many notices showing up for some sites. And I've seen questions in Slack from this more than once where the solution was turning WP_DEBUG off. :shrug:

**Comment by curtisbelt** [comment]
@brandoncamenisch 
Yup tested through with the build script :slightly_smiling_face:.
Good call on jq... would a "Dependencies" section in the README suffice? Or how would you usually handle that?

**Comment by ethitter** [comment]
Superseded by #10

**Pull request closed without merging**